### PR TITLE
Migrate AWS manifests to v3 pattern

### LINKED ISSUE #1095: v3 version of AWS KFDefs
See: #1062 
AWS KFDefs should follow v3 patterns in order to not depend on kfctl kustomize magic

/cc @Jeffwan

### PR DESCRIPTION
**Pull request by Jeffwan**
**Which issue is resolved by this Pull Request:**
Resolves #1095

**Description of your changes:**
Get some time to migrate AWS manifest to v3 pattern.  I will have an update kfdef PR coming later. 

**Checklist:**
- [x] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate-changed-only`
    3. `make test`

### CODE REVIEW

**Comment by jlewi** [comment]
@Jeffwan Is there a co-maintainer of the AWS manifests who could review this PR? Maybe @ellistarn ?

**Comment by Jeffwan** [comment]
@jlewi  @PatrickXYS will review this PR but we don't have permission to leave approve since this is a new folder. He will leave lgtm after review. It adds a new OWNER files under stacks/aws and we can approve ourselves later.

File: aws/istio-ingress/base_v3/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```
[the remark this answers is not in the document]
Comment (Jeffwan) [reply aws/istio-ingress/base_v3/kustomization.yaml:2]: yeah. I think we should explicitly list here. Will do

**Comment by PatrickXYS** [comment]
Seems like we haven't updated files like `kfctl_aws.yaml`

**Comment by Jeffwan** [comment]
> es like `kfctl_aws.yaml`

It could be in a separate PR. That's not related.

**Pull request merged**